frontend/src/components/Pages/MissionPage/VideoStream/VideoStreamWindow.tsx

```diff
@@ -40,7 +40,7 @@ export function VideoStreamWindow({ videoStreams }: VideoStreamWindowProps) {
         toggleFullScreenMode()
     }
 
-    const videoCards = videoStreams.map(function (videoStream, index) {
+    const videoCards = videoStreams.map((videoStream, index) => {
         return (
```

Comment:
The { and the "return" is not needed here, but I suppose this is not the focus of this refactoring anyways